x-pack/plugins/observability_solution/observability_ai_assistant_management/public/routes/components/search_connector_tab.tsx

```diff
@@ -16,7 +16,7 @@ export const SELECTED_CONNECTOR_LOCAL_STORAGE_KEY =
 
 export function SearchConnectorTab() {
   const { application } = useKibana().services;
-  const url = application.getUrlForApp('enterprise_search', { path: '/content/connectors' });
+  const url = application.getUrlForApp('enterpriseSearch', { path: '/content/connectors' });
```

Comment:
Am I wrong or this URL was broken?

Reply:
This was briefly broken last week in the O11y Solution nav because enterprise search was disabled, but we fixed that and the link worked when I checked. But you're right that the appId should be `enterpriseSearch` so that is very confusing 🤔

Reply:
I confirm this is fixed externally to this PR too (noticed when rebasing).